ui/App.svelte

```diff
@@ -76,5 +79,8 @@
 <NotificationFaucet style="margin-top: calc(var(--topbar-height) + 11px)" />
 <Theme />
 <Remote {store} let:data={session} context="session">
-  <Router {routes} />
+  <TransactionCenter />
```

Comment:
I cleared my session and now this gets stuck on the `Sidebar` trying to render a `null` `identity`; might need to revisit how the layout renders before the `identity` exists.